utils/jose/Makefile

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=jose
+PKG_VERSION:=7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://github.com/latchset/$(PKG_NAME)/releases/download/v$(PKG_VERSION)/
+PKG_HASH:=2bb1d228944de62140403aacb874fe2cfbfbf979d6362fedb39427f1b1d2a702
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+CFLAGS += -fhonour-copts 
+TARGET_CFLAGS += -std=gnu99 -O3 $(FPIC)
+TARGET_LDFLAGS += -Wl,-rpath-link=$(1)/usr/lib
+
+MAKE_FLAGS += LD="$(TARGET_CC) -o"
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/jose
+  SECTION:=utils
+  TITLE:=Provides a full crypto stack including key generation, signing and encryption.
+  DEPENDS:=+zlib +jansson +libopenssl +libpthread
+  URL:=https://github.com/latchset/jose
+  MAINTAINER:=Tibor Dudlák <tibor.dudlak@gmail.com>
+endef
+
+define Package/jose/description
+	José is a command line utility for performing various tasks on JSON
+	Object Signing and Encryption (JOSE) objects. José provides a full
+	crypto stack including key generation, signing and encryption.
+endef
+
```

Comment:
Please put the CFLAGS, TARGET_CFLAGS , TARGET_LDFLAGS and MAKE_FLAGS variables here